notebooks/quickstart/data-scientist/02-explore-dataset.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Let's now select the family age dataset from the list of available datasets.</p><p> </p><p>Now, let's view the selected Family Age dataset. Every available dataset will have a name, description and a list of assets attached to it.</p><p>We can also see the shape and dtype of each asset attached to this dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>